app/src/main/java/com/example/musictoned/addExercise/AddExerciseRoute.kt

```diff
@@ -10,7 +10,7 @@ import androidx.lifecycle.viewmodel.compose.viewModel
 
 @Composable
 fun AddExerciseRoute(
-    onNavigateToEditRoutine: () -> Unit
+    onNavigateToEditRoutine: (exerciseName: String) -> Unit
```

Comment:
Same question here, should we consolidate the naming to be consistent? e.g., routineName